FlowCrypt/Info.plist

```diff
@@ -79,5 +79,24 @@
 		<string>UIInterfaceOrientationLandscapeRight</string>
 		<string>UIInterfaceOrientationPortraitUpsideDown</string>
 	</array>
+    <key>NSAppTransportSecurity</key>
+    <dict>
+        <key>NSPinnedDomains</key>
+        <dict>
+            <key>flowcrypt.com</key>
+            <dict>
+                <key>NSIncludesSubdomains</key>
+                <true/>
+                <key>NSPinnedLeafIdentities</key>
```

Comment:
I tried to send password-protected messages, but there is no such option on compose screen because `isUsingFes` is `false`.

I think it happens because app can't fetch correct configuration as here for root certificates we should use `NSPinnedCAIdentities` instead of `NSPinnedLeafIdentities`.